contracts/libs/0.8.x/BytecodeStorageV1.sol

```diff
@@ -141,26 +180,28 @@ library BytecodeStorage {
             // store length of data in first 32 bytes
             mstore(data, size)
             // copy code to memory, excluding the deployer-address
-            extcodecopy(_address, add(data, 0x20), DATA_OFFSET, size)
+            extcodecopy(_address, add(data, 0x20), dataOffset, size)
         }
     }
 
     /**
      * @notice Get address for deployer for given contract bytecode
-     * @param _address address of deployed contract with bytecode containing concat(invalid opcode,  deployer-address, data)
+     * @param _address address of deployed contract with bytecode containing concat(invalid opcode, version, deployer-address, data)
```

Comment:
nit: perhaps a comment here noting that V0 formatting is also accepted.

Reply:
Good call out – updated the documentation of this to be both clearer and more maintainable.